packages/wrangler/src/dev/miniflare.ts

```diff
@@ -381,6 +382,82 @@ function buildSitesOptions({ assetPaths }: ConfigBundle) {
 	}
 }
 
+function handleRuntimeStdio(stdout: Readable, stderr: Readable) {
+	// ASSUMPTION: each chunk is a whole message from workerd
+	// This may not hold across OSes/architectures, but it seems to work on macOS M-line
+	// I'm going with this simple approach to avoid complicating this too early
+	// We can iterate on this heuristic in the future if it causes issues
+	const classifiers = {
+		// Is this chunk a big chonky barf from workerd that we want to hijack to cleanup/ignore?
+		isBarf(chunk: string) {
+			const containsLlvmSymbolizerWarning = chunk.includes(
+				"Not symbolizing stack traces because $LLVM_SYMBOLIZER is not set"
+			);
+			const containsHexStack = /stack:( [a-f\d]{9}){3,}/.test(chunk);
+
+			return containsLlvmSymbolizerWarning || containsHexStack;
+		},
+		// Is this chunk an Address In Use error?
+		isAddressInUse(chunk: string) {
+			return chunk.includes("Address already in use; toString() = ");
+		},
+	};
+
+	stdout.on("data", (chunk: Buffer | string) => {
+		chunk = chunk.toString();
+
+		if (classifiers.isBarf(chunk)) {
+			// this is a big chonky barf from workerd that we want to hijack to cleanup/ignore
+
+			// CLEANABLE:
+			// there are no known cases to cleanup yet
+			// but, as they are identified, we will do that here
+
+			// IGNORABLE:
+			// anything else not handled above is considered ignorable
+			// so send it to the debug logs which are discarded unless
+			// the user explicitly sets a logLevel indicating they care
+			logger.debug(chunk);
+		}
+
+		// anything not exlicitly handled above should be logged as info (via stdout)
+		else {
+			logger.info(chunk);
```

Comment:
I was aware of this because miniflare's previous implementation used `readline` which stripped the newline chars before using console.log/error but adding .trim() to the chunk values didn't change anything